rules/do-you-use-the-best-exception-handling-library/rule.md

```diff
@@ -16,38 +16,50 @@ guid: 4758ac66-d4a5-4ccd-93cc-85c1f2d369da
 
 ::: greybox
 **Application Insights** will tell you if your application goes down or runs slowly under load. If there are any uncaught exceptions, you’ll be able to drill into the code to pinpoint the problem. You can also find out what your users are doing with the application so that you can tune it to their needs in each development cycle.
 :::
 
-If Application Insights is not available we use Seq when developing web applications. From its [page](https://datalust.co/seq):
+Application Insights gives you very useful graphs and analysis which give you a good overview of how things are going. See [here](/rules-to-better-application-insights) for more details.
 
-::: greybox
-**Seq** is built for modern structured logging with message templates. Rather than waste time and effort trying to extract data from plain-text logs with fragile log parsing, the properties associated with each log event are captured and sent to Seq in a clean JSON format. Message templates are supported natively by ASP.NET Core, Serilog, NLog, and many other libraries, so your application can use the best available diagnostic logging for your platform.
+::: good
+![Figure: Good - Application Insights gives you graphs and analysis that help you find issues, but also lets you drill down to get the details as well](overview.png)
 :::
 
-[Raygun](https://raygun.com) is another great tool as it helps you identify and monitor errors in Single Page Applications.
+If Application Insights is not available, we use [**Seq**](https://datalust.co/seq) when developing web applications. Seq is great for identifying specific issues and how to fix them, but is not as good as Application Insights at letting you see the big picture.
 
-Application Insights gives you very useful graphs and analysis which give you a good overview of how things are going. See [here](/rules-to-better-application-insights) for more details. Seq is great for identifying specific issues and how to fix them, but is not as good at letting you see the big picture.
+::: greybox
+**Seq** is built for modern structured logging with message templates. Rather than waste time and effort trying to extract data from plain-text logs with fragile log parsing, the properties associated with each log event are captured and sent to Seq in a clean JSON format. Message templates are supported natively by ASP.NET Core, Serilog, NLog, and many other libraries, so your application can use the best available diagnostic logging for your platform.
+:::
 
 ::: good
 ![Figure: Good - Seq provides you with plenty of details about what is happening, but if you don't already know what you're looking for, it can be tricky to parse](xn4QHnmBS0Kx39gOv0wM_GettingStarted-1.png)
 :::
 
-::: good
-![Figure: Good - Application Insights gives you graphs and analysis that help you find issues, but also lets you drill down to get the details as well](overview.png)
+::: greybox
```

Comment:
instead of greyboxes I'd make them headings using ##

Reply:
Used headings instead of the grey boxes :)